package.json

```diff
@@ -92,18 +93,24 @@
     "@popperjs/core": "^2.11.8",
     "@tippyjs/react": "^4.2.6",
     "@types/lodash": "^4.14.202",
+    "@types/yargs": "^17.0.32",
```

Comment:
Nit: I think the @types can be dev deps, so consumers don't need to install them.